### DISCUSSION TITLE
Insert stop-overs en-route to destination

### DISCUSSION THREAD

**Discussion by pooky2483** [albar965/littlenavmap#810] [Ideas]
I have created a route from EGNX to YMER and as it's a long flight which I would like to do in a small aircraft, I have selected airports in 500 mile steps to land at (to take a break) and then take off to continue the flight. 
I have found that I can select the airports, but I am unable to set as arrival as when I do that, I see a flight path from the final destination to the airport I just selected.
Would it be possible to be able to insert airports into a route AND be able to set  an arrival from the previous airport and departure path going on to the next airport and so-on all the way to the final airport.

**Comment by albar965** [comment]
Sorry, intermediate stops are not doable with reasonable effort. You can create a big master flight plan and then save the snippets from the flight plan table using [this function](https://www.littlenavmap.org/manuals/littlenavmap/release/2.6/en/FLIGHTPLAN.html#save-selected-range-as-flight-plan).
Alex

**Comment by pooky2483** [comment]
I know it's not possible ATM, but could it be considered to be implemented as a feature.

**Reply by albar965** [reply]
Whenever I have an idea to do this in an easy way, I'd to it. But for now I don't, sorry.
I keep in in mind.

Also, no GPS, FMS, MCDU or whatever is used in the simulators won't be able to deal with this anyway.
